src/voodoo/VoodooClient.ts

```diff
@@ -256,6 +236,40 @@ export default class VoodooClient {
     return new TextDecoder().decode(jsonAsUtf8Bytes)
   }
 
+  /**
+   * Retrieves all the user contact bundles for a given peer address
+   */
+  async getUserContactMultiBundle(
+    peerAddress: string
+  ): Promise<VoodooMultiBundle | undefined> {
+    const stream = this.apiClient.queryIterator(
+      { contentTopic: buildVoodooUserContactTopic(peerAddress) },
+      { pageSize: 25, direction: SortDirection.SORT_DIRECTION_DESCENDING }
+    )
+
+    // Get a list of all valid contacts
+    const listContacts: VoodooContact[] = []
+
+    for await (const env of stream) {
+      if (!env.message) {
+        continue
+      }
+      // TODO: need to use more than just the public JSON, need to define a proto or class
+      // that includes a signature etc
+      const voodooPublicJson = await this.decodeEnvelopeRaw(env)
+
+      // TODO: do validation here of the address signature
+      const voodooInstance =
+        this.wasm.addOrGetPublicAccountFromJSON(voodooPublicJson)
+      listContacts.push(new VoodooContact(peerAddress, voodooInstance))
+    }
+    return {
+      address: peerAddress,
+      contacts: listContacts,
+      timestamp: new Date().getTime(),
```

Comment:
Do we actually need a timestamp here? Given that the multi-bundle spans a time range, doesn't seem particularly useful (even if we used the timestamp of the first or last contact).